build_system/src/build.rs

```diff
@@ -55,8 +56,9 @@ impl BuildArg {
     }
 }
 
+/// Builds the sysroot for the specified environment and configuration.
```

Comment:
Please remove this comment as it is a duplicate of the code itself, so it doesn't bring anything more than the code.